wolfcrypt/src/aes.c

```diff
@@ -6996,6 +6996,14 @@ int wc_AesCcmEncrypt(Aes* aes, byte* out, const byte* in, word32 inSz,
             || authTag == NULL || nonceSz < 7 || nonceSz > 13)
         return BAD_FUNC_ARG;
 
+    /* sanity check on tag size */
```

Comment:
Let's put this code into a function since it's used multiple times.  Also, please document the supported tag size values.